retrofit2-clients/src/main/java/com/palantir/remoting1/retrofit2/Retrofit2ClientBuilder.java

```diff
@@ -50,15 +52,27 @@ public Retrofit2ClientBuilder(ClientConfig config) {
     @Override
     public <T> T build(Class<T> serviceClass, String userAgent, List<String> uris) {
         Preconditions.checkArgument(!uris.isEmpty());
-        okhttp3.OkHttpClient client = createOkHttpClient(userAgent, uris);
+        List<String> sanitizedUris = sanitizeUris(uris); // adds trailing slash
+        okhttp3.OkHttpClient client = createOkHttpClient(userAgent, sanitizedUris);
         Retrofit retrofit = new Retrofit.Builder()
                 .client(client)
-                .baseUrl(uris.get(0))
+                .baseUrl(sanitizedUris.get(0))
                 .addConverterFactory(JacksonConverterFactory.create())
                 .build();
         return retrofit.create(serviceClass);
     }
 
+    private static List<String> sanitizeUris(List<String> uris) {
```

Comment:
I'd rather call this addTrailingSlash, as it doesn't do any other sanitization. Also, you wouldn't need a comment above to clarify as `// adds trailing slash`